components/gripper/trossen/trossen.go

```diff
@@ -3,58 +3,53 @@ package trossen
 
 import (
 	"context"
-	"sync"
-	"time"
+	"errors"
 
 	"github.com/edaniels/golog"
-	"github.com/jacobsa/go-serial/serial"
-	"go.uber.org/multierr"
-	"go.viam.com/dynamixel/network"
-	"go.viam.com/dynamixel/servo"
-	"go.viam.com/dynamixel/servo/s_model"
-	"go.viam.com/utils"
 
+	"go.viam.com/rdk/components/arm"
+	trossenarm "go.viam.com/rdk/components/arm/trossen"
 	"go.viam.com/rdk/components/generic"
 	"go.viam.com/rdk/components/gripper"
 	"go.viam.com/rdk/config"
-	"go.viam.com/rdk/operation"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/registry"
 	"go.viam.com/rdk/resource"
 	rdkutils "go.viam.com/rdk/utils"
 )
 
+// This is an implementation of the Gripper API for Trossen arms. The gripper
+// requires a single Trossen arm component in its dependencies upon configuration,
+// as all of the gripper functions defer to DoCommand on the arm implementation. See
+// components/arm/trossen/trossen.go for more about the arm implementations
+// and supported arms
+
 var (
 	modelNameWX250s = resource.NewDefaultModel("trossen-wx250s")
 	modelNameVX300s = resource.NewDefaultModel("trossen-vx300s")
 )
 
-// AttrConfig is the config for a trossen gripper.
+// AttrConfig is the config for a trossen gripper. WARNING: These
+// attributes no longer do anything and should be removed in a
+// later commit - GV.
 type AttrConfig struct {
-	SerialPath string `json:"serial_path"`
-	BaudRate   int    `json:"serial_baud_rate"`
+	SerialPath string `json:"serial_path,omitempty"`
+	BaudRate   int    `json:"serial_baud_rate,omitempty"`
 }
 
 // Validate ensures all parts of the config are valid.
 func (config *AttrConfig) Validate(path string) error {
```

Comment:
Assuming we go the route of having the arm as a config attribute, add the arm as an implicit dependency